app/src/main/java/com/droidhats/campuscompass/repositories/MapRepository.kt

```diff
@@ -204,12 +204,42 @@ class MapRepository(applicationContext: Context) {
             "Faubourg Building" -> R.drawable.building_fg
             "Visual Arts Building" -> R.drawable.building_va
             "Pavillion J.W. McConnell Building" -> R.drawable.building_webster_library
+            "Grey Nuns Building" -> R.drawable.building_grey_nuns
+            "Samuel Bronfman Building" -> R.drawable.building_sb
+            "GS Building" -> R.drawable.building_gs
+            "CB Building" -> R.drawable.building_cbb
```

Comment:
This should be  R.drawable.building_cb
(remove the last 'b' letter)

Reply:
Thanks, I've updated the resource name 😁